scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -385,6 +385,14 @@
   com.netflix.fenzo.VirtualMachineLease
   (cpuCores [_] (or (offer-resource-scalar offer "cpus") 0.0))
   (diskMB [_] (or (offer-resource-scalar offer "disk") 0.0))
+  (getScalarValue [_ name] (or (double (offer-resource-scalar offer name)) 0.0))
+  (getScalarValues [_]
+    (reduce (fn [result resource]
+                 (if-let [value (:scalar resource)]
+                   (update-in result [(:name resource)] (fnil + 0.0 0.0) value)
```

Comment:
Can you add a comment about it then. Anyone that looks at it will be confused, try to remove it and then get bit.